src/metrics_server/README.md

```diff
@@ -23,6 +23,7 @@ The metrics server supports two URL paths:
         userId: string,
         countries: string[],
         bytesTransferred: number,
+        tunnelTimeMs: number,
```

Comment:
Let's use seconds. It's consistent with the metrics and Milliseconds is too fine-grained.